pkg/backend/telemetryapi/harvester.go

```diff
@@ -257,14 +290,15 @@ func postData(req *http.Request, client *http.Client) response {
 	if resp.StatusCode == http.StatusOK || resp.StatusCode == http.StatusAccepted {
 		r.body, _ = ioutil.ReadAll(resp.Body)
```

Comment:
Should we cover err case?

Reply:
This was from the original code found at [here](https://github.com/newrelic/newrelic-telemetry-sdk-go/blob/master/telemetry/harvester.go#L203). Not yet gone in to change this but agree we should check for an error. I'd raise this as a new issue though (will be interesting to test).